src/ex3/index.js

```diff
@@ -0,0 +1,73 @@
+import fetch from "node-fetch"
+import { Command } from "commander"
+import PokemonClient from "./PokemonClient.js"
+import fs from "fs"
+
+const pokemonClient = new PokemonClient()
+
+async function getPokemon(id) {
+  const isInteger = parseInt(id)
+  if (isInteger) {
+    const response = await fetch(`${pokemonClient.api}${id}`)
```

Comment:
all of the previous features have been erased! (ability to catch multiple pokemon at once for example)